app/controllers/alchemy/admin/attachments_controller.rb

```diff
@@ -7,7 +7,7 @@ class AttachmentsController < ResourcesController
       include ArchiveOverlay
 
       add_alchemy_filter :by_file_type, type: :select,
-        options: -> { Alchemy::Attachment.file_types(_1.result) }
+        options: ->(_q) { Alchemy::Attachment.file_types }
```

Comment:
This allows to pre-select the File Type select and always shows all file types instead of filtering the file types by the search quer (what does not make any sense in this use case anyway)

Reply:
rework done